README.md

```diff
@@ -72,3 +72,40 @@ cd src/redux/api
 npm install
 npm start
 ```
+
+## System description
+
+| Component   | Description  | Accessible by |
+| ----------- | :----------- | :-----------  |
+| Slide | A slide is the visible content on a screen. | Admin |
+| Media | Media is either images or videos used as content for slides. | Admin, editor |
+| Theme | A theme has css, that can override the slide css. | Admin |
+| Template | The template is how the slide looks, and which content is on the slide | ? |
```

Comment:
Templates are imported from manually and developed outside the admin. Templates are accessible to choose on Slides. 
Accessible by: Admin, editor.